test/load/packer/loadtest-ami/loadtest.pkr.hcl

```diff
@@ -0,0 +1,51 @@
+packer {
```

Comment:
We should update the [`./test/load/packer/README.md`](https://github.com/hashicorp/consul/tree/7f3c3b093221f62779281012eb9ac0e26774c7da/test/load/packer#load-test-ami) to point to this new file.